notebooks/01.2 IPython Numpy and Matplotlib Refresher.ipynb

```diff
@@ -37,13 +88,23 @@
    "source": [
     "import numpy as np\n",
     "\n",
+    "# Setting a random seed for reproducibility\n",
```

Comment:
This is not really the recommended way, right? Would you do it like this? That can be tricky, I think.